Tools/GNUMake/comps/cray.mak

```diff
@@ -13,26 +13,18 @@ F90FLAGS =
 
 ########################################################################
 
-# CRAY_CC_VERSION is defined by the 'cce' module. Starting with CCE 9, Cray
-# changed the C and C++ compilers to clang/LLVM based, so all of the
-# options/flags for those compilers changed. But the Cray Fortran compiler is
-# still based on CCE 8 and so it has the same options as before.
-COMP_VERSION = $(shell echo $(CRAY_CC_VERSION) | cut -f 1 -d .)
-
-ifeq ($(shell expr $(COMP_VERSION) \>= 9), 1)
-  CCE_GE_V9 := TRUE
+ifneq ($(shell CC --version | grep "LLVM"),)
+  CRAY_IS_CLANG_BASED = TRUE
 else
-  CCE_GE_V9 := FALSE
+  CRAY_IS_CLANG_BASED = FALSE
 endif
 
 # we need to be careful on the A64fx
-ifeq ($(CRAY_CC_VERSION),10.0.1)
-  ifeq ($(CRAY_ARCH_TARGET),aarch64)
-    CCE_GE_V9 := FALSE
-  endif
+ifeq ($(CRAY_ARCH_TARGET),aarch64)
```

Comment:
Why is this needed, in addition to the grep check above? Wouldn't this be false anyway on Ookami?

Reply:
Oh I forgot to remove the old check.